modules/programs/jujutsu.nix

```
@@ -22,6 +22,14 @@ in {
 
     package = mkPackageOption pkgs "jujutsu" { };
 
+    ediff = mkOption {
+      type = types.bool;
+      default = config.programs.emacs.enable;
```

Unfortunately I think you'll need a `defaultText` here to avoid making the documentation depend on the configuration:

```suggestion
      default = config.programs.emacs.enable;
      defaultText = literalExpression "config.programs.emacs.enable";
```